parser.go

```diff
@@ -97,18 +100,48 @@ func SetMarkdownFileDirectory(directoryPath string) func(*Parser) {
 	}
 }
 
-// ParseAPI parses general api info for gived searchDir and mainAPIFile
+// ParseAPI parses general api info for given searchDir and mainAPIFile
 func (parser *Parser) ParseAPI(searchDir string, mainAPIFile string) error {
 	Println("Generate general API Info")
+
 	if err := parser.getAllGoFileInfo(searchDir); err != nil {
 		return err
 	}
 
+	cmd := exec.Command("go", "list", "-f={{.ImportPath}}")
+	cmd.Dir = searchDir
+	var stdout, stderr strings.Builder
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("execute go list command, %s", err)
+	}
+
+	outStr, _ := stdout.String(), stderr.String()
+	if outStr[0] == '_' {
```

Comment:
What is the situation like this?

Reply:
go list command will show like `_/{GOPATH}/src/{YOUR_PACKAGE}` when NOT enable GO MODULE.

Reply:
I want to write comment.
What do you think?

Reply:
yep, I will add comment for it.